source/style-guide/style/punctuation.txt

```diff
@@ -138,7 +138,7 @@ in the *Chicago Manual of Style*.
        Unlike the other alarms in this list you set the network check
        alarm variable upon network check creation.
 
-       To learn more, see Upgrading Ops Manager.
+       To learn more see Upgrading Ops Manager.
```

Comment:
The comma belongs here.

```suggestion
       To learn more, see Upgrading Ops Manager.
```

Reply:
See [Use a comma to separate an introductory word, phrase, or clause from the rest of the sentence.](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#commas:~:text=Use%20a%20comma%20to%20separate%20an%20introductory%20word%2C%20phrase%2C%20or%20clause%20from%20the%20rest%20of%20the%20sentence.) and the related example.